R/decisiontreeutils.R

```diff
@@ -0,0 +1,431 @@
+#' Error checking pre-model
+#' Does not return anything - just throws errror
+#'
+#' @param config list of config options
+#' @param the.data incoming data
+preModelCheckDT <- function(config, the.data) {
+  data_names <- names(the.data)
```

Comment:
Function names are better as verbs. This could be renamed to something like `check...`.